packages/angular_devkit/build_angular/src/utils/index-file/style-nonce.ts

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { htmlRewritingStream } from './html-rewriting-stream';
+
+/**
+ * Pattern matching the name of the Angular nonce attribute. Note that the it is
+ * case-insensitive, because HTML attribute names are case-insesitive as well.
+ */
```

Comment:
```suggestion
/**
 * Pattern matching the name of the Angular nonce attribute. Note that this is
 * case-insensitive, because HTML attribute names are case-insensitive as well.
 */
```